pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -408,6 +408,10 @@ func getDiskTargetsForMigration(dom cli.VirDomain, vmi *v1.VirtualMachineInstanc
 	}
 	// the name of the volume should match the alias
 	for _, disk := range disks {
+		// explicitly skip cd-rom drives
+		if disk.Device == "cdrom" {
+			continue
+		}
 		if disk.ReadOnly != nil && !migrationVols.isGeneratedVolume(disk.Alias.GetName()) {
```

Comment:
What I find interesting: I see here `disk.ReadOnly`: Do we have an explanation why the readonly cdrom drive was not skipped by this?